nucliadb/src/nucliadb/common/maindb/pg.py

```diff
@@ -107,6 +107,11 @@ async def delete(self, key: str) -> None:
             async with self.connection.cursor() as cur:
                 await cur.execute("DELETE FROM resources WHERE key = %s", (key,))
 
+    async def delete_batch(self, keys: list[str]) -> None:
```

Comment:
This could be combined with `keys()` to do it in a single query like: `cur.execute('DELETE FROM resources WHERE key LIKE %s', (prefix + '%',))` (and passing the prefix into this, rather than the list of keys). In the case of the local storage, it could move the iteration over keys inside the method or do a `rmdir`

Reply:
thanks! addressed